kobo/device.go

```diff
@@ -74,7 +74,7 @@ const (
 
 // Cover types.
 const (
-	CoverTypeLibFull CoverType = "N3_FULL"
+	CoverTypeLibFull CoverType = "N3_LIBRARY_FULL"
```

Comment:
IIRC, actually, it's both.

`N3_FULL` is the actual screen-sized one (sleep screen), while `N3_LIBRARY_FULL` is the one used in the Details view.

Reply:
Are the sizes the same, or do I need to add another set of sizes?

Reply:
Nevermind, I just realized I was diverging from my notes when I changed those around. I'll fix the sizes.